src/modules/sdk/objections/service.ts

```diff
@@ -14,6 +14,9 @@ const OBJECTIONS_API_PATCH_URL = (companyNumber: string, objectionId: string): s
 const OBJECTIONS_API_ADD_ATTACHMENT_URL = (companyNumber: string, objectionId: string): string =>
   OBJECTIONS_API_URL(companyNumber) + `/${objectionId}/attachments`;
 
+const OBJECTIONS_API_GET_ATTACHMENTS_LIST_URL = (companyNumber: string, objectionId: string): string =>
+    OBJECTIONS_API_URL(companyNumber) + `/${objectionId}/attachments`;
```

Comment:
May be better as it is identical to OBJECTIONS_API_ADD_ATTACHMENT_URL to combine them under the name OBJECTIONS_API_ATTACHMENT_URL

Reply:
Updated